package-lock.json

```diff
@@ -15,6 +15,7 @@
         "d3-selection": "3.0.0",
         "d3-zoom": "3.0.0",
         "font-awesome": "4.7.0",
+        "highlight.js": "^11.10.0",
```

Comment:
chore: Pin the version on the lockfile as well

Reply:
https://github.com/HathorNetwork/hathor-explorer/pull/293/commits/40aba4857a7fec3680eddd1f6365dee1c2766d82